bioconda_utils/pkg_test.py

```diff
@@ -0,0 +1,104 @@
+import subprocess as sp
+import tempfile
+import tarfile
+import os
+import shlex
+
+from conda_build.metadata import MetaData
+
+'mulled-build build-and-test "gffutils" --test "python -c \'import gffutils\''
+
+
+
+
+
+def get_tests(path):
+    "Extract tests from a built package"
+    tmp = tempfile.mkdtemp()
+    t = tarfile.open(path)
+    t.extractall(tmp)
+    input_dir = os.path.join(tmp, 'info', 'recipe')
+
+    tests = []
+    recipe_meta = MetaData(input_dir)
+
```

Comment:
I didn't want to mess around with building a fake `args` object to use the existing mulled-build test extraction, so just wrote one here that takes a package. While it would be nice to just import from mulled_build, it might take some refactoring on your end which I'm not sure is worth it.